### PR TITLE
Fixed url links in valkey.conf

### PR DESCRIPTION
**Pull request by PingXie**

### CODE REVIEW

File: valkey.conf [hunk 1]
```diff
@@ -329,7 +329,7 @@ daemonize no
 #
 # When the server runs non daemonized, no pid file is created if none is
 # specified in the configuration. When the server is daemonized, the pid file
-# is used even if not specified, defaulting to "/var/run/redis.pid".
+# is used even if not specified, defaulting to "/var/run/valkey.pid".
```
Comment (zuiderkwast) [inline valkey.conf]: This is not an URL.

The default is controlled by a macro in server.h, which is still defined like this:

    #define CONFIG_DEFAULT_PID_FILE "/var/run/redis.pid"

The comments here better match the actual default.

**Review by zuiderkwast — DISMISSED** [review]
LGTM except for the non-URL.

File: valkey.conf (hunk 1 above)
Comment (madolson) [reply valkey.conf]: Yes, this is the only remaining reference to redis that should exist in the valkey.conf file.

Comment (zuiderkwast) [reply valkey.conf]: We mentioned that changing this default might be considered a breaking change, so we'll do it in the next major version. I created this now: #321.

Comment (madolson) [inline valkey.conf]:
```suggestion
# is used even if not specified, defaulting to "/var/run/redis.pid".
```

**Review by madolson — APPROVED** [review]

**Comment by madolson** [comment]
I keep forgetting if I push a commit I have to add the DCO :/

File: valkey.conf (hunk 1 above)
Comment (PingXie) [reply valkey.conf]: Thanks for the reminder, @zuiderkwast

Comment (PingXie) [reply valkey.conf]: Thanks for catching this one too @madolson. Apparently I am OOM'ing on all the compat decisions ...

**Comment by Katherine199119** [comment]
Fackk

**Pull request merged**